_data/projects/nodatime.yml

```diff
@@ -0,0 +1,12 @@
+name: Noda Time
+desc: A better date/time API for .NET
+site: https://github.com/nodatime/nodatime
+tags:
+- C#
```

Comment:
Would .NET be useful here as well? It's not clear whether the .NET and C# tags are effectively equivalent...